packages/manager/modules/web-paas/src/details/service/edit-name/edit-name.html

```diff
@@ -0,0 +1,24 @@
+<form id="projectName" data-ng-submit="$ctrl.edit()">
```

Comment:
**question**
Hmm is there a reason that the main action isn't set directly on the `oui-modal` ? (WIth `primary-action`)